### ISSUE TITLE
List index out of range

### ISSUE THREAD

**Issue by tpaktop77** [freenas/py-SMART#15]
```# python
Python 2.7.5 (default, Aug  4 2017, 00:39:18)
[GCC 4.8.5 20150623 (Red Hat 4.8.5-16)] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> from pySMART import DeviceList
>>> devlist = DeviceList()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/root/virtenvs/pysmart/lib/python2.7/site-packages/pySMART/device_list.py", line 51, in __init__
    self._initialize()
  File "/root/virtenvs/pysmart/lib/python2.7/site-packages/pySMART/device_list.py", line 106, in _initialize
    self.devices.append(Device(name))
  File "/root/virtenvs/pysmart/lib/python2.7/site-packages/pySMART/device.py", line 140, in __init__
    self.update()
  File "/root/virtenvs/pysmart/lib/python2.7/site-packages/pySMART/device.py", line 461, in update
    ASCQ = line_[2][:-1]
IndexError: list index out of range
```

**Comment by tpaktop77** [comment]
This is partial SMART output:

```
SMART Self-test log
Num  Test              Status                 segment  LifeTime  LBA_first_err [SK ASC ASQ]
     Description                              number   (hours)
# 1  Background short  Completed                   -     176                 - [-   -    -]
# 2  Background short  Aborted (by user command)   -     111                 - [-   -    -]
# 3  Background short  Aborted (by user command)   -      86                 - [-   -    -]
# 4  Background short  Aborted (by user command)   -      86                 - [-   -    -]
# 5  Background short  Aborted (by user command)   -      86                 - [-   -    -]
# 6  Background short  Aborted (by user command)   -      86                 - [-   -    -]
# 7  Background short  Aborted (by user command)   -      86                 - [-   -    -]
# 8  Background short  Aborted (by user command)   -      85                 - [-   -    -]
# 9  Background short  Aborted (by user command)   -      60                 - [-   -    -]
#10  Background short  Aborted (by user command)   -      35                 - [-   -    -]
#11  Background short  Aborted (by user command)   -       1                 - [-   -    -]
#12  Background short  Aborted (by user command)   -       1                 - [-   -    -]
#13  Background short  Aborted (by user command)   -       1                 - [-   -    -]
#14  Background short  Aborted (by user command)   -       1                 - [-   -    -]
#15  Background short  Aborted (by user command)   -       1                 - [-   -    -]
Long (extended) Self Test duration: 15620 seconds [260.3 minutes]
```

**Comment by william-gr** [comment]
This library is no longer under development. If you have a PR we will incorporate but we will not investigate issues. Thanks.

**Comment by ophilli** [comment]
It looks like this issue was fixed in pr #16 - is it possible to close this issue now?
